client/src/pages/Application/AdminView/Step6.js

```diff
@@ -31,9 +32,12 @@ const Column = styled.div`
 function FosterAndDogInformation() {
   return (
     <Column>
-      <Meetings
-        title="Foster  Profile"
-        // TODO
+      <FosterProfile
+        name="Amy C."
+        ambassadorName="Kristin"
+        coordinatorName="Andy L."
+        fosterHistory={[{ name: "Lolita" }, { name: "Flower" }]}
+        internalNotes="Lorem ipsum internal notes here"
```

Comment:
Pass in data from the backend once you have Parth's stuff merged into your branch